src/main/java/me/ajaja/module/remind/application/model/RemindStrategyFactory.java

```diff
@@ -20,7 +20,7 @@ public List<SendRemindPort> getAllStrategies() {
 	}
 
 	public SendRemindPort get(String remindType) {
-		if (remindType.equals("KAKAO") || remindType.equals("BOTH")) {
+		if ("KAKAO".equals(remindType) || "BOTH".equals(remindType)) {
 			return alimtalkStrategy;
 		}
 		return emailStrategy;
```

Comment:
심플하게 가져가신다면 이렇게도 해볼 수 있겠군요~?!
```suggestion
		return "EMAIL".equals(remindType) ? emailStrategy : alimtalkStrategy;
```

Reply:
말씀하신대로 적용하였습니다 ㅎ